editions/tw5.com/tiddlers/concepts/CoordinateSystems.tid

```diff
@@ -0,0 +1,42 @@
+created: 20220810201659784
+modified: 20220810201659784
+tags: Concepts
+title: Coordinate Systems
```

Comment:
I wonder if it would be helpful to rename this tiddler to Popup Coordinate System. It should also be linked to from the tiddler "PopupMechanism".

Reply:
Hello @saqimtiaz, thank you for taking the time to review my PR.
Are the coordinates really limited to be used with popups? I would prefer a more generic name for the documentation because these coordinates could be used for other things as well. Currently they are used for popups but should we suggest in the documentation, that they are limited to this use case in the future?